Description: don't call my_init()
 Since MySQL 8.0.2, my_init() isn't exported anymore and shouldn't have to be
 called by clients directly. See
 https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-2.html
Author: Andreas Hasenack <andreas@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/apache2/+bug/1859773
Forwarded: no
X-Forwarded-Reason: Should have an explicit check for MySQL 8.0.2
Last-Update: 2020-03-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/dbd/apr_dbd_mysql.c
+++ b/dbd/apr_dbd_mysql.c
@@ -1263,9 +1263,6 @@
 
 static void dbd_mysql_init(apr_pool_t *pool)
 {
-#if MYSQL_VERSION_ID < 100000
-    my_init();
-#endif
     mysql_thread_init();
 
     /* FIXME: this is a guess; find out what it really does */
